src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -116,35 +116,37 @@ export const pageContent = {
       'I understand that if I act contrary to the specification, I am liable to be slashed.',
   },
   [AcknowledgementIdsEnum.keyManagement]: {
-    title: 'Keys safety',
+    title: 'Keys management',
     content: (
-      <Text size="large" className="my10">
-        Your validator keys are derived from a unique mnemonic (or seed). Your
-        seed will be the ONLY WAY to withdraw your funds. Above all, keep it
-        safe!
-      </Text>
-    ),
-    acknowledgementText:
-      'I understand that my mnemonic (seed) will be the ‼️ONLY WAY‼️ to withdraw my funds.',
-  },
-  [AcknowledgementIdsEnum.signingKeys]: {
-    title: 'Signing keys',
-    content: (
-      <Text size="large" className="my10">
-        We'll help you create keys for each of your validators. These keys will
-        be saved in key-stores and you will need to give these to your validator
-        software to begin validating. You will also receive a deposit file to
-        upload to this website with the public keys for your validator.
-      </Text>
+      <>
+        <Text size="medium" className="mt10">
+          To become a validator you'll need to know about managing keys. If
+          you're unfamiliar with keys and mnemomics, please don't proceed. Your
+          keys will be your responsibility and no one can help you if you lose
+          them.
+        </Text>
+        <Text size="medium" className="mt20">
+          Your validator keys are derived from a unique mnemonic (or seed). Your
+          seed will be the ONLY WAY to withdraw your funds. Above all, keep it
+          safe!
+        </Text>
+        <Text size="medium" className="mt20">
+          We'll help you create keys for each of your validators. These keys
+          will be saved in key-stores and you will need to give these to your
+          validator software to begin validating. You will also receive a
+          deposit file to upload to this website with the public keys for your
+          validator.
+        </Text>
+      </>
     ),
     acknowledgementText:
-      'I will safeguard my key-stores and give them to my validator software to start validating.',
+      'I understand that keys are my responsibility and that my mnemonic (seed) will be the ‼️ONLY WAY‼️ to withdraw my funds.',
```

Comment:
```suggestion
      'I understand that my keys are my responsibility and that my mnemonic (seed) will be the ‼️ONLY WAY‼️ to withdraw my funds.',
```

Reply:
WE should come up with an alternative to `‼️ONLY WAY‼️` - I could see that creating confusion for translators. e.g. we could put this within bold HTML elements now that we know how.

Reply:
Sounds reasonable. As long as it remains accentuated as this is different to how keys and accounts work on eth1.